### ISSUE TITLE
Error while deploying Volume Mount sample from aspire-samples

### ISSUE THREAD

**Issue by adamtrip** [prom3theu5/aspirational-manifests#87]
### 🔥 Bug Description
```
(?) Done: Decrypting secrets from ./aspirate-output

Executing: kubectl apply -k
C:\Users\Adam\source\repos\aspire-samples\samples\VolumeMount\VolumeMount.AppHost\aspirate-output against kubernetes
context kubernetes-admin@kubernetes.
error: accumulating resources: accumulation err='accumulating resources from 'sqlserver':
'C:\Users\Adam\source\repos\aspire-samples\samples\VolumeMount\VolumeMount.AppHost\aspirate-output\sqlserver' must
resolve to a file': recursed accumulation of path
'C:\Users\Adam\source\repos\aspire-samples\samples\VolumeMount\VolumeMount.AppHost\aspirate-output\sqlserver':
accumulating resources: accumulating resources from 'sqlserver.yml': MalformedYAMLError: yaml: line 21: did not find
expected '-' indicator in File: sqlserver.yml
Failed to deploy manifests in
'C:\Users\Adam\source\repos\aspire-samples\samples\VolumeMount\VolumeMount.AppHost\aspirate-output'

(?) Done: Deployments successfully applied to cluster 'kubernetes-admin@kubernetes'

 ?? Execution Completed
```

### 🔍 Steps to Reproduce the Bug
1. Clone aspirate-samples Volume Mount project
2. Add secret for the sqlserver password `dotnet user-secrets set sqlpassword <password>`
3. Run `aspire init`
4. Run `aspire generate`
5. Run `aspire apply`

**Comment by adamtrip** [comment]
here's the yaml file that it generated, might be helpfull:
```
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: sqlserver-configuration
  labels:
    app: sqlserver
data:
  ACCEPT_EULA: "Y"
---
apiVersion: v1
kind: Secret
metadata:
  name: sqlserver-secrets
labels:
  app: sqlserver
type: Opaque
data:
  MSSQL_SA_PASSWORD: Qzc0Y24tXlUwUmswZ2djcyZRbiVQdkA4
---
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: sqlserver-statefulset
  labels:
    app: sqlserver
spec:
  serviceName: "sqlserver"
  replicas: 1
  selector:
    matchLabels:
      app: sqlserver
  template:
    metadata:
      labels:
        app: sqlserver
    spec:
      containers:
        - name: sqlserver
          image: mcr.microsoft.com/mssql/server:2022-latest
          envFrom:
            - configMapRef:
                name: sqlserver-configuration
            - secretRef:
            name: sqlserver-secrets
          ports:
            - containerPort: 1433
              name: sqlserverdb
---
apiVersion: v1
kind: Service
metadata:
  name: sqlserver-service
  labels:
    app: sqlserver
spec:
  type: ClusterIP
  ports:
    - port: 1433
      name: sqlserver
  selector:
    app: sqlserver
```

**Comment by prom3theu5** [comment]
Cheers Adam
Fix for this just gone out with [v0.1.21-preview](https://github.com/prom3theu5/aspirational-manifests/tree/v0.1.21-preview)
